android/build.gradle

```diff
@@ -18,10 +18,13 @@ rootProject.allprojects {
     repositories {
         google()
         mavenCentral()
+        // just a test
+        maven { url 'https://storage.googleapis.com/download.flutter.io' }
     }
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'org.jetbrains.kotlin.android'
```

Comment:
Is this necessary for us to use Kotlin? If no, let's remove this.

Reply:
Yes this is required. This compiles the kotlin code to the JVM bytecode.